README.md

```diff
@@ -49,6 +49,58 @@ variations and defines a default region.
 
 The above query would also produce the same output.
 
+# Development
+
+For ease of use, instead of installing
+[pgxn-client](https://pgxn.github.io/pgxnclient/) locally, you can use the
+provided docker container.
+
+* Start the container (it will stay up for 10 days by default):
+
+```sh
+docker-compose up -d
+```
+
+* Enter the container:
+
+```sh
+docker-compose exec pgxn-tools bash
+```
+
+* Then you can enter the PostgreSQL instance:
+
+```sh
+psql -U postgres -d contrib_regression
+```
+
+* Create the extension:
+
+``` sql
+create extension holidays;
```

Comment:
@madtibo This command doesn't works too because extension is not installed on database I think we should add:

```sh
# make install
```

before no ?

Reply:
@Its-Alex That's right. We need to install the library after the instance creation (since it install the binaries while at it).